recipes/settings.py

```diff
@@ -555,4 +555,6 @@
 ACCOUNT_EMAIL_SUBJECT_PREFIX = os.getenv(
     'ACCOUNT_EMAIL_SUBJECT_PREFIX', '[Tandoor Recipes] ')  # allauth sender prefix
 
+ENABLE_EXTERNAL_CONNECTORS = bool(int(os.getenv('ENABLE_EXTERNAL_CONNECTORS', False)))
```

Comment:
we have new docs for the configuration parameters, can you add this setting to the list there. Also I think we can enable it by default, it looks good so far.